db/migrate/20231129095416_create_users.rb

```diff
@@ -4,6 +4,8 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
+      t.datetime :updated_at
+      t.datetime :created_at
```

Comment:
- [ ] [optional] Kindly consider removing these two lines, the  ` t.timestamps` will automatically create these two columns.

Reply:
@danmainah Thank you so much for reminding me. I appreciate